explain/grid.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 'The pganalyze EXPLAIN grid view'
+backlink_href: /docs/explain
+backlink_title: 'Documentation: EXPLAIN - Overview'
+---
+
+import imgGrid from './grid.png'
+
+export const ImgGrid = () => <img src={imgGrid} alt="Plan Grid Example" />
+
+The pganalyze EXPLAIN grid view is an alternate way to inspect query plans. It's
+more compact, and modeled after the native Postgres text EXPLAIN format, but
+includes some quality of life improvements.
+
+<ImgGrid />
+
+Note that this view is a work in progress, and may not include all the plan
+information you're interested in yet. In that case, check the Node Tree EXPLAIN
+view or the source of individual nodes (and please [contact us with
+feedback](mailto:support@pganalyze.com)).
+
+## Plan layout
+
+The query plan is displayed in a way similar to the Postgres text EXPLAIN format,
+but with some important differences:
+
+ - Common Table Expressions (WITH clause entries) and any other independent
+   subqueries are broken out into their own section.
+ - The focus is on the shape of the plan and node details are available only
+   after clicking to select a specific node.
+ - Scan details include the table being scanned, the index used (if any), and
+   the alias used for this table reference in the query (if any).
+
+
+## Metrics
+
+The execution metric to show can be selected at the top of the plan:
+
+  - **Est. Cost**: The cost of the node according to the Postgres planning model,
+    excluding the cost of the node's children.
+  - **Runtime**: How much time was spent executing this node, excluding time spent
+    executing the node's children.
+  - **Rows**: How many rows each node produced, and how many it was expected to
+    produce according to the query plan
+  - **Buffers**: How many buffers were hit in the buffer cache and read from disk
+    (these are combined) or dirtied while executing this query, excluding
+    buffers touched while executing the node's children
+  - **Reads**: How much data was read from disk (or the OS page cache) and how much
```

Comment:
Likely not a question to ask here but I was curious - when track_io_timing is off, how does this look like?

Reply:
It should be `-`, but I think we actually show zeros; I'll fix that separately. We should also maybe show why: it's not that hard for us to check the setting.